### DISCUSSION TITLE
Realtime connection closes and realtime.subscription table is empty

### DISCUSSION THREAD

**Discussion by kavitang** [supabase/supabase#11973] [Questions] [answered]
Hey,

I'm trying to set up a `table-db-changes` channel on a table in Supabase. All steps done, table is set up in Replication with insert notifications enabled. This was working correctly when I first tried it, but it has stopped working since. The connection is successfully established but closes for some reason
![Realtime_Close1](https://user-images.githubusercontent.com/96101675/215089422-85c82dd7-153c-45cf-9f78-a2220af6d458.jpeg)
![Realtime_Leave3](https://user-images.githubusercontent.com/96101675/215089445-c31f6aa7-ece2-4476-a2f9-28ee4174754e.jpeg)
![Notifications_WS1](https://user-images.githubusercontent.com/96101675/215089552-a9a67842-d6b9-4544-83eb-dd96bd6f7452.jpeg)

I can see an entry in the `realtime.subscription` table for a few seconds and then it's gone. I'm not sure what is happening here or how to resolve this. I already have another channel with presence up and running correctly on a different channel, that has not  been affected.  Should I reuse the same channel? The one change that happened was I renamed the table, could this have caused an issue?

Note: I'm setting up two different channels for presence and realtime postgres changes.

```
// Postgres changes
 const channel = supabaseClient
      .channel("table-db-changes")
      .on(
        "postgres_changes",
        {
          event: "INSERT",
          schema: "public",
          table: "user_notifications",
        },
        (payload) => {
          console.log(payload);
        }
      )
      .subscribe();

// Presence
supabaseClient.channel("channel_name", {
      config: {
        presence: {
          key: userId,
        },
      },
    });
```

Things I have tried (unsuccessfully) :
- Restarted the server
- Tried with a brand new table
- Toggled RLS policies on and off
- Setting event to `*` instead of `INSERT`

I would appreciate any help here as this is key feature for us.

**Comment by GaryAustin1** [comment]
Any errors in the realtime logs on the dashboard?
Double check the table itself has replication on.
You cutoff your websocket messages, so if there was an error there it does not show.

**Comment by kavitang** [comment, marked as answer]
@GaryAustin1 - There are no errors since the last 24 hours, but before that I can see several, though none of these appeared the WS messages.
```
Event Message
Subscribing to PostgreSQL failed: {:error, {:subscription_insert_failed, %{"event" => "*", "schema" => "public", "table" => "*"}}}
```
But for some reason it has started working now. I'm not sure what triggered it to work again, I made no changes to the actual connection code.. 🤔
